ciscoise_connector.py

```diff
@@ -91,9 +111,11 @@ def _call_ers_api(self, endpoint, action_result, data=None, allow_unknown=True,
 
         return phantom.APP_SUCCESS, ret_data
 
-    def _call_rest_api(self, endpoint, action_result, schema=None, data=None, allow_unknown=True):
+    def _call_rest_api(self, endpoint, action_result, schema=None, data=None, allow_unknown=True, try_secondary_device=False):
```

Comment:
Instead of using the `try_secondary_device` flag, how about we use a required variable? Something like, base_url. When we call the method for the second time, then instead of passing try_secondary_device, we can simply pass `base_url=self. _secondary_device` and for the first call we can pass `base_url=self. _base_url`.